### DISCUSSION TITLE
Custom contest with PDF certificate generation

### DISCUSSION THREAD

**Discussion by fooons** [magicbug/Cloudlog#1560] [Ideas / Feature Requests]
Hello.

I would like to suggest a new feature for CloudLog that could benefit activators and radio clubs that organize activities whose goal is to get a certain score or simply a certain number of contacts.
The idea is to be able to create personalized "contests/activities" in which a PDF certificate is obtained if the objectives have been met. A small ranking could be included.
I understand that it is a function that is not useful for everyone, but it would be interesting  to expand this web application.

**Comment by magicbug** [comment]
Nice idea but beyond the scope of a general logging applicaiton

**Reply by fooons** [reply]
I agree with you.
It goes beyond a conventional log and I hope that in the future CloudLog will be a web application reference for radio amateurs and its use will expand to Radio Clubs.
I don't have much experience with Codeigniter, but I'm going to take a look at the documentation to try to contribute and try to materialize this idea.
